pandas/io/tests/test_pickle.py

```diff
@@ -32,6 +33,7 @@ class TestPickle():
     """
     _multiprocess_can_split_ = True
 
+    @pytest.fixture(autouse=True)
```

Comment:
don't use ``autouse`` its a bit magical, create explict fixtures (we can come back and fix this later though if easier)